packages/mattermost-redux/src/client/client4.ts

```diff
@@ -3120,6 +3120,13 @@ export default class Client4 {
         );
     };
 
+    getPrevTrialLicense = () => {
+        return this.doFetch<StatusOK>(
```

Comment:
I think the `StatusOK` should be changed to the returned data type. I believe the backend does not return StatusOK but rather information which should be the proper type here

Reply:
Fixed.